src/nvim/eval.c

```diff
@@ -15000,19 +15000,6 @@ static void get_system_output_as_rettv(typval_T *argvars, typval_T *rettv,
     // res may contain several NULs before the final terminating one.
     // Replace them with SOH (1) like in get_cmd_output() to avoid truncation.
     memchrsub(res, NUL, 1, nread);
-#ifdef USE_CRNL
```

Comment:
We need that for Windows, it sets the use of Carrier Return Line Feed. Don't remove.